ern-local-cli/src/index.ts

```diff
@@ -189,6 +201,15 @@ const logLevelStringToEnum = (level: string) => {
       ? new KaxNullRenderer()
       : new KaxAdvancedRenderer(kaxRendererConfig)
 
+  const nodeVersion = process.version
+  if (!isNodeVersionCompatible(nodeVersion)) {
+    const requiredVersion = getNodeEngineVersionRequirement()
+    log.error(`This version of ern requires Node ${requiredVersion}
+Node version currently in use is ${nodeVersion}
+Please switch to a version of Node satisfying the version requirement`)
```

Comment:
Minor: If you still need to rebase, you can update the text to the full name `Node.js` instead of `Node`.